frontend/public/components/routes.tsx

```diff
@@ -133,6 +209,11 @@ const TLSSettings = props => <span>
 const RouteDetails: React.SFC<RoutesDetailsProps> = ({obj: route}) => <React.Fragment>
   <div className="co-m-pane__body">
     <ResourceSummary resource={route} showPodSelector={false} showNodeSelector={false}>
+      <dt>Hostname</dt>
+      <dd><RouteHostname obj={route} />
```

Comment:
I think the value under hostname should be just the hostname, and we should add the URL to the top of the details page.